telegram/utils/helpers.py

```diff
@@ -19,6 +19,10 @@
 """This module contains helper functions."""
 from html import escape
 
+try:
+    from urllib import quote  # noqa: F401
```

Comment:
I'm guessing F401 is `unused import`. But it seems unused. Why is it imported?

Reply:
Very good question.... removed :)